src/compiler/checker.ts

```diff
@@ -91,6 +93,9 @@ module ts {
         var globalBooleanType: ObjectType;
         var globalRegExpType: ObjectType;
         var globalTemplateStringsArrayType: ObjectType;
+        var globalPromiseType: ObjectType;
+        var globalIPromiseType: ObjectType;

```

Comment:
The "I" prefix feels like more of a C# convention. Maybe globalPromiseInterface or something like that